src/SFA.DAS.EmployerFinance/Models/LevyDeclarationSagaType.cs

```diff
@@ -0,0 +1,8 @@
+namespace SFA.DAS.EmployerFinance.Models
+{
+    public enum LevyDeclarationSagaType : short
+    {
+        All = 0,
```

Comment:
We don't strictly need the initializers.

Reply:
Same as above.